eZ/Publish/Core/Persistence/Cache/BookmarkHandler.php

```diff
@@ -93,4 +93,17 @@ public function countUserBookmarks(int $userId): int
 
         return $this->persistenceHandler->bookmarkHandler()->countUserBookmarks($userId);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function locationSwapped(int $location1Id, int $location2Id): void
+    {
+        $this->logger->logCall(__METHOD__, [
+            'location1Id' => $location1Id,
+            'location2Id' => $location2Id,
+        ]);
+
+        $this->persistenceHandler->bookmarkHandler()->locationSwapped($location1Id, $location2Id);
```

Comment:
Maybe add a note here saying cache clearing is effectively done by locationHandler, to explain why we don't clear cache here.